modules/uninstalling-olm-using-ansible.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * install_config/installing-operator-framework.adoc
+
+[id='uninstalling-olm-using-ansible_{context}']
+= Uninstalling Operator Lifecycle Manager using Ansible
+
+To uninstall the Technology Preview Operator Framework, you can use the included
+playbook with the {product-title} `openshift-ansible` installer after installing
+your cluster.
+
+.Prerequisites
+
+- An existing {product-title} 3.11 cluster
+- Access to the cluster using an account with `cluster-admin` permissions
+- Ansible playbooks provided by the latest `openshift-ansible` installer
+
+.Procedure
+
+. Add the following variables to your `config.yml` playbook:
```

Comment:
Is this supposed to be `config.yml` or `config.yaml`? Just asking because it doesn't look like it likes the source,yaml variables you have here in the preview.

Reply:
It's valid for YAML files to use the `.yml` extension, so it's likely that this is correct.

Reply:
Ack